app/models/manageiq/providers/workflows/automation_manager/workflow.rb

```diff
@@ -1,4 +1,26 @@
 class ManageIQ::Providers::Workflows::AutomationManager::Workflow < ManageIQ::Providers::EmbeddedAutomationManager::ConfigurationScriptPayload
+  def self.seed
+    manager = ManageIQ::Providers::Workflows::AutomationManager.in_my_region.first
+    return if manager.nil?
+
+    repo = manager.configuration_script_sources.find_by(:name => "Embedded Workflows")
+    return if repo.nil?
+
+    to_delete = repo.configuration_script_payloads.pluck(:id)
+    Vmdb::Plugins.embedded_workflows_content.each do |filename|
+      relative_filename = filename.split("content/workflows/").last
```

Comment:
What happens if 2 plugins provide the same workflow name?  Wondering if we should straight disallow that through specs, or perhaps we prefix the name with the plugin name?

Reply:
Super minor, but Pathname does this for you...

```suggestion
      relative_filename = Pathname.new(filename).relative_path_from("content/workflows")
```

Reply:
I updated the core method to yield Pathnames but I don't think `relative_path_from` works like this at least not for absolute paths
```
(byebug) filename.relative_path_from("content/workflows")
*** ArgumentError Exception: different prefix: "/" and "content/workflows"

nil
(byebug) filename
#<Pathname:/home/grare/adam/src/manageiq/manageiq-providers-workflows/content/workflows/hello_world/workflow.asl>
```